tests/http/config/caching.graphql

```diff
@@ -0,0 +1,20 @@
+schema @upstream(baseURL: "http://example.com", batch: {delay: 1, maxSize: 1000}) {
+  query: Query
+}
+
+type Query {
+  foos: [Foo] @http(path: "/foos")
+  bars: [Bar] @http(path: "/bars")
+}
+
+type Foo @cache(maxAge: 10) {
+  id: Int! @cache(maxAge: 15)
+  name: String! @cache(maxAge: 15)
+  bar: Bar @http(path: "/bars", query: [{key: "fooId", value: "{{value.id}}"}], groupBy: ["fooId"])
+}
+
+type Bar @cache(maxAge: 5) {
+  id: Int! @cache(maxAge: 10)
+  fooId: Int! @cache(maxAge: 20)
+  foo: [Foo] @http(path: "/foos", query: [{key: "id", value: "{{value.fooId}}"}], groupBy: ["id"])
+}
```

Comment:
seems like we can specify `@cache` both on types and on fields. How will it work in case we specify also cache here for `foo`, considering it's specified for the type `Foo`?

Reply:
the `@cache` provided at the type level will take affect for all the fields inside `Foo` but for the field `foo` the value provided at field level will take affect. I'll do some tests and to get a clearer picture of what's happening